test/pkey_meth_kdf_test.c

```diff
@@ -195,8 +221,13 @@ static int test_kdf_scrypt(void)
 
 int setup_tests(void)
 {
-    ADD_TEST(test_kdf_tls1_prf);
-    ADD_TEST(test_kdf_hkdf);
+    int tests = 1;
+
+    if (fips_provider_version_ge(NULL, 3, 3, 0))
```

Comment:
After 3.3 is released this number will need to be 3,4,0

Reply:
Will it? Assuming the fix gets into 3.3.0, then any version greater-or-equal to that version of the FIPS provider will also have the fix. It should not be necessary to update it.

But if 3.3.0 does not include this (perhaps 3.3.1 does?) then it will need to be modified. I would hate to have to have a range of values here, especially in the test code.

Reply:
I have no idea if this will make it into 3.3, since this is at beta stage now.

Reply:
This is a bug fix, so it will be backported to all branches. I am not sure if we want to check all possible versions here as that would make the version check very complex. Although it might be a good idea to add a helper function that would make these bug fix release version checks more simple.

I do not think it will make it to 3.3.0 as the release is imminent. It should make it to 3.3.1.

Reply:
I will note that this is in _test_ code. If that makes a difference.